components/TextFields/src/MDCMultilineTextField.m

```diff
@@ -474,13 +474,6 @@ - (void)updateBorder {
 
 #pragma mark - Properties Implementation
 
-#if defined(__IPHONE_10_0) && (__IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_10_0)
```

Comment:
Since all this is leftover from when we tried to keep the superclass as UITextView, go ahead and delete adjusts and setAdjusts also.

Reply:
Done